composables/useIsChain.ts

```diff
@@ -8,6 +8,9 @@ export default function (prefix: ComputedRef<Prefix>) {
   const isRemark = computed(
     () => prefix.value === 'rmrk' || prefix.value === 'ksm',
   )
+
+  const isRmrk = computed(() => prefix.value === 'rmrk')
+
```

Comment:
what's the diff?

Reply:
between?

royalty amount input should appear for all except rmrk